### PR TITLE
Adm 609 add document of encrypt decrypt configuration and change document calculating pipeline file name

### PR DESCRIPTION
**Pull request by BoBoDai**
…k document

## Summary

...

## Before

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## After

_Description_

**Screenshots**
If applicable, add screenshots to help explain behavior of your code.

## Note

_Null_

### CODE REVIEW

File: docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+layout: ../../../layouts/MainLayout.astro
+---
+
+# Spike -- Encrypt & Decrypt configuration
+
+## Document link
+
+Due to confidentiality reasons, spike documents are only accessible to internal personnel.
+
+[Encrypt & Decrypt configuration](https://docs.google.com/document/d/1Drdi8sXX37QdKFdOJB3WgFfjrBNRfNTH/edit)
```
Comment (guzhongren) [inline docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md:13]: Please `as doc` here, not link to other website

**Review by guzhongren — COMMENTED** [review]
user really wants to use our tool as we designed?

File: docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md (hunk 1 above)
Comment (BoBoDai) [reply docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md:13]: hello，zhongren，Due to the encryption and decryption process being only visible to internal personnel, so only the Google Drive link is placed in the docs

Comment (guzhongren) [reply docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md:13]: Hey, @BoBoDai 
as an open-source tool, should keep every thing open.
So, Covert the google doc as code here.
It is very important to store the `diagram as code` here

Comment (BoBoDai) [reply docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md:13]: Okay, I may need to ask the security team first, and then we can discuss it.

Comment (davidwangcn9) [reply docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md:13]: Hi @BoBoDai , after discussed with someone from security, I think @guzhongren is correct. Keep it open and put the document here is okay. There are a lot of open source encryptions outside. It doesn't mean they are insecure. So keep it open should be a good way.

**Review by davidwangcn9 — CHANGES_REQUESTED** [review]

File: docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md (hunk 1 above)
Comment (BoBoDai) [reply docs/src/pages/en/dev-center/tech-spikes-encrypt-decrypt-configuration.md:13]: ok,i got it

**Comment by guzhongren** [comment]
hey @BoBoDai where is the sequence diagram listed in the google doc?

**Pull request merged**